gui/default/syncthing/folder/folderFilesModalView.html

```diff
@@ -0,0 +1,27 @@
+<modal id="folderFiles" status="default" icon="fas fa-cog" heading="{{'Included files' | translate}}" large="yes" closeable="yes">
+    <div class="modal-body">
```

Comment:
Breadcrumbs are quite a good idea. I'll see what I can do.

Did you mean to have `..` instead of the back button?
> users are used to that
Did you mean users of command line? 😀 IMHO, not all of the users know command line, but all of them surely know a file browser. All of the file browsers have a back button. (At least I think so, not sure.)
Or did you just mean to put it on the top of the panel instead of the bottom?

Reply:
I don't really care if it's back arrow as a list item, or .., or a back button above the item list, but usually these controls are at the top, not at the bottom.

Reply:
Ah yes, now I understand.